### PR TITLE
Improve/config

### PR DESCRIPTION
**Pull request by KdVos**

### CODE REVIEW

File: src/jsonconfig.h [hunk 1]
```diff
@@ -101,16 +106,28 @@ class Config{
...
-    void print(){
+    void print(){// todo add custom spawn location
         std::cout << "Simulator configuration settings:" << std::endl;
         std::cout << "Uncertain odom: " << uncertain_odom.value() << std::endl;
         std::cout << "Show full map: " << show_full_map.value() << std::endl;
         std::cout << "disable_speedcap: " << disable_speedcap.value() << std::endl;
         std::cout << "enable taco: " << enable_taco.value() << std::endl;
         std::cout << "use pyro: " << use_pyro.value() << std::endl;
+        std::cout << "Spawn Location: "<< spawn.value().getOrigin()<< std::endl;
+        std::cout << "Spawn Rotation: " << spawn.value().getBasis() << std::endl; // Todo get yaw, geolib implm gives unexpected results.
```
Comment (PetervDooren) [inline src/jsonconfig.h:130]: I think geolib rotations have a getRPY function for this.

File: src/main.cpp [hunk 2]
```diff
@@ -290,8 +291,10 @@ int main(int argc, char **argv){
         }
 
         // Visualize
+        bool full_map = config.show_full_map.value();
+              
         if (visualize)
-            visualization::visualize(world, robots, collision, config.show_full_map.value(),bbox, robot_radius);
+            visualization::visualize(world, robots, collision, full_map, bbox, robot_radius);
```
Comment (PetervDooren) [inline src/main.cpp:297]: This change does not add much. I would revert this

**Review by PetervDooren — CHANGES_REQUESTED** [review]
Some minor beauty points but the implementation itself looks good to me

File: src/jsonconfig.h [hunk 3]
```diff
@@ -101,16 +106,28 @@ class Config{
...
+        }
+        else{
+            spawn = geo::Pose3D::identity(); // [x,y,theta] = [0,0,0]
+        }
+        
 
     }
 
-    void print(){
+    void print(){// todo add custom spawn location
```
[the remark this answers is not in the document]
Comment (KdVos) [reply src/jsonconfig.h:122]:
```suggestion
    void print(){
```

File: src/jsonconfig.h (hunk 1 above)
Comment (KdVos) [reply src/jsonconfig.h:130]: Correct. There is a getYaw function to get the yaw from a Pose3D. This function, however, doesn't behave as I would expect. I've made an issue in geolib, to investigate what happens on that side. 

https://github.com/tue-robotics/geolib2/issues/62#issue-1247648600

**Pull request merged**